pkg/driver/mysql/mysql.go

```diff
@@ -145,6 +145,9 @@ func (drv *Driver) mysqldumpArgs() []string {
 	if password, set := drv.databaseURL.User.Password(); set {
 		args = append(args, "--password="+password)
 	}
+	if socket := drv.databaseURL.Query().Get("socket"); drv.databaseURL.Hostname() == "" && socket != "" {
```

Comment:
I forget, do we have any tests for this function? Would be nice to add a test to cover the new functionality.

Reply:
Currently, no. But I might try to come up with some testing. Let me see what I can do :-)